components/list/list.js

```diff
@@ -0,0 +1,60 @@
+// Imports
+import React, { PropTypes } from 'react';
+import { getClassNamesWithMods } from '../_helpers';
+
+/**
+ * General List component. Use when you need to display array of elements
+ */
+function List({ items, hideBullets, mods = [], align }) {
+  mods.push(`align_${align}`);
```

Comment:
this would have a side-effect. especially if the `mods` array is passed as a reference from the top-level.

just saying 😬